src/components/@shared/atoms/Table/index.tsx

```diff
@@ -3,18 +3,23 @@ import DataTable, { IDataTableProps } from 'react-data-table-component'
 import Loader from '../Loader'
 import Pagination from '@shared/Pagination'
 import styles from './index.module.css'
-import { useUserPreferences } from '@context/UserPreferences'
 
 export interface TableProps extends IDataTableProps {
   isLoading?: boolean
   emptyMessage?: string
   sortField?: string
   sortAsc?: boolean
   className?: string
+  chainIds: number[]
 }
 
-function Empty({ message }: { message?: string }): ReactElement {
-  const { chainIds } = useUserPreferences()
+function Empty({
+  message,
+  chainIds
```

Comment:
we should not deal with any `chainIds` in here, this component should not be concerned about any of that and it is only used to construct a different `emptyMessage`. so all this logic should be moved over to wherever we need that 'No network selected' message